src/components/CustomPressable/index.js

```diff
@@ -19,7 +19,7 @@ const CustomPressable = (props) => {
       onPress={onPress}
       style={style}
       android_ripple={{
-        color: COLORS.gray9,
+        color: colors.gray1,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value for the android ripple effect has been changed from `COLORS.gray9` to `colors.gray1`. Make sure this change is intentional and that `colors.gray1` is the desired color for the ripple effect. If not, revert it back to the original color.
```diff
- color: COLORS.gray9,
+ color: colors.gray1,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->